RELEASES.md

```diff
@@ -1,3 +1,122 @@
+Version 1.37.0 (2019-08-15)
+==========================
+
+Language
+--------
+- [`#[must_use]` will now warn if the type is in a tuple and unused.][61100]
+- [`#[must_use]` will now warn if the type is in a `Box` and unused.][62228]
+- [`#[must_use]` will now warn if the type is in a array and unused.][62235]
```

Comment:
```suggestion
- [`#[must_use]` will now warn if the type is in an array and unused.][62235]
```

Reply:
Consider merging the 3 must_use changes and link to the prs on mentioning Box, arrays, and tuples.